Makefile

```diff
@@ -228,9 +227,25 @@ add-license: addlicense
 
 # Build the OCI image
 .PHONY: oci-build
-oci-build: generate manifests manager test-envtest
+oci-build: manifests generate fmt vet test-envtest
 	BUILDAH_FORMAT=docker $(IMAGE_BUILDER) build -t $(OPERATOR_IMG) .
 
+# PLATFORMS defines the target platforms for the manager image to provide support to multiple
+# architectures. (i.e. make docker-buildx OPERATOR_IMG=quay.io/cryostat/cryostat-operator:2.3.0).To use this option you need to:
+# - use docker as IMAGE_BUILDER.
+# - able to use docker buildx . More info: https://docs.docker.com/build/buildx/
+# - have enable BuildKit, More info: https://docs.docker.com/develop/develop-images/build_enhancements/
+# To properly provided solutions that supports more than one platform you should use this option.
+PLATFORMS ?= linux/arm64,linux/amd64,linux/s390x,linux/ppc64le
+.PHONY: docker-buildx
+docker-buildx: manifests generate fmt vet test-envtest ## Build OCI image for the manager for cross-platform support
+	# copy existing Dockerfile and insert --platform=${BUILDPLATFORM} into Dockerfile.cross, and preserve the original Dockerfile
+	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' Dockerfile > Dockerfile.cross
+	- docker buildx create --name project-v3-builder
+	docker buildx use project-v3-builder
+	- docker buildx build --platform=$(PLATFORMS) --tag $(OPERATOR_IMG) -f Dockerfile.cross .
```

Comment:
```bash
$ podman buildx --help
Build images

Description:
  Build images

Usage:
  podman buildx [command]

Aliases:
  buildx, builder

Available Commands:
  build       Build an image using instructions from Containerfiles
  prune       Remove unused images
```

There is a `podman buildx` for cross-compilation too - is it not sufficient for this case? I'm not sure what this `create` and `use` is for, but building for a different platform with a specific tag on a given Dockerfile should all work.

Reply:
yehh those `create` and `use` is the headache cuz podman does not support that. But from reading this doc: https://docs.docker.com/engine/reference/commandline/buildx_create/,

> All Docker contexts also get the default builder instance.

I think we can just remove them to be compatible with `podman`?

Reply:
I think so. Worth trying it out, anyway. In my experience with doing cross-platform builds with podman, all I needed to do was use `--platform` and install the `qemu-user-static` RPM package. I didn't need to create any temporary build contexts etc.

Reply:
I am seeing this without switching to another builder:

```
ERROR: multiple platforms feature is currently not supported for docker driver. 
Please switch to a different driver (eg. "docker buildx create --use")
make: *** [Makefile:245: oci-buildx] Error 1
```
We really need to create one then.